modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/init/ResourceManagerInitStrategy.java

```diff
@@ -5,15 +5,5 @@
  */
 package dev.galasa.framework.internal.init;
 
-import dev.galasa.framework.Framework;
-import dev.galasa.framework.spi.FrameworkException;
-import dev.galasa.framework.spi.IConfigurationPropertyStoreService;
-
 public class ResourceManagerInitStrategy extends DefaultInitStrategy {
```

Comment:
Why do we no longer need this method? Assuming we need to keep this class even though it no longer does anything just to not break any other classes that return it?